dataflow/gemma/custom_model_gemma.py

```diff
@@ -50,6 +50,10 @@ def __init__(
         self._env_vars = {}
 
     def share_model_across_processes(self) -> bool:
+        """ Indicates if the model should be loaded once-per-worker rather than
+        once-per-thread on a worker. Because Gemma is a large language model,
+        this will always return True to avoid OOM errors.
```

Comment:
```suggestion
        """ Indicates if the model should be loaded once-per-VM rather than
        once-per-worker-process on a VM. Because Gemma is a large language model,
        this will always return True to avoid OOM errors.
```

Guessing this will need linting help/feel free to reword, but the current wording is incorrect